TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationScreen.tsx

```diff
@@ -93,6 +102,17 @@ const CalibrationScreen: React.FC<CalibrationScreenProps> = ({ isHandPresent, on
         };
     }, []);
 
+    useEffect(() => {
```

Comment:
Could this be in CalibrationManager instead?

Reply:
Moved :+1: